spec/features/admin/products/products_spec.rb

```diff
@@ -522,7 +522,7 @@ def build_option_type_with_values(name, values)
         end
       end
 
-      context 'with limited permissions' do
+      xcontext 'with limited permissions' do
```

Comment:
@damianlegawiec This test looks like it belongs in Vendo, not here, what do you think?